webapp/templates/partials/scoreboard_table.html.twig

```diff
@@ -114,6 +119,21 @@
             {% set previousTeam = null %}
         {% endif %}
 
+        {# process awards color #}
+        {% set awardColor = '' %}
+        {% if showLegends and processAwards and score.team.category.name in awardsCategories %}
```

Comment:
Later we can add an option whether the award numbers are global (i.e. we give 4 total gold awards) or per category (i.e. we give 4 gold awards per category). For now this is the most useful behavior so I like it.

Also is the `name` needed here? Not sure

Reply:
It seems that `awardsCategories` is an array of categories' name

Reply:
I wonder how. Anyway if it works it's fine. I will investigate further after merging.

Reply:
Actually you are right. I just tested for it, team categories' name is not needed here, thank you.